typescript/widgets/src/walletIntegrations/multiProtocol.tsx

```diff
@@ -40,11 +40,14 @@ const logger = widgetLogger.child({
   module: 'walletIntegrations/multiProtocol',
 });
 
+type NonStarknetProtocol = Exclude<ProtocolType, 'starknet'>;
+type ProtocolRecord<T> = Record<NonStarknetProtocol, T>;
```

Comment:
`ProtocolMap` might be a better word given the existing `ChainMap` nomenclature we use